app/start.ps1

```diff
@@ -0,0 +1,60 @@
+Write-Host ""
+Write-Host "Loading azd .env file from current environment"
+Write-Host ""
+
+foreach ($line in (& azd env get-values)) {
+    if ($line -match "([^=]+)=(.*)") {
+        $key = $matches[1]
+        $value = $matches[2] -replace '^"|"$'
+        Set-Item -Path "env:\$key" -Value $value
+    }
+}
+
+if ($LASTEXITCODE -ne 0) {
+    Write-Host "Failed to load environment variables from azd environment"
+    exit $LASTEXITCODE
+}
+
+
+Write-Host 'Creating python virtual environment "backend/backend_env"'
+python -m venv backend/backend_env
+
+Write-Host ""
+Write-Host "Restoring backend python packages"
+Write-Host ""
+Set-Location backend
+./backend_env/scripts/pip3 install -r requirements.txt
```

Comment:
This is only going to work on windows. Which is fine for the hooks. Not sure how much we care about non-windows users running the PowerShell scripts outside the hook, but if we do, we also have to check the `backend_env/bin` folder...

Reply:
We should be consistent in using `python -m pip` (from the virtual environment) to install packages